pkg/pgmodel/sql_reader.go

```diff
@@ -86,15 +86,14 @@ func (q *pgxQuerier) LabelsCacheCapacity() int {
 }
 
 // entry point from our own version of the prometheus engine
-func (q *pgxQuerier) Select(mint int64, maxt int64, sortSeries bool, hints *storage.SelectHints, path []parser.Node, ms ...*labels.Matcher) (storage.SeriesSet, parser.Node, storage.Warnings, error) {
+func (q *pgxQuerier) Select(mint int64, maxt int64, sortSeries bool, hints *storage.SelectHints, path []parser.Node, ms ...*labels.Matcher) (storage.SeriesSet, parser.Node) {
 	rows, topNode, err := q.getResultRows(mint, maxt, hints, path, ms)
-
 	if err != nil {
-		return nil, nil, nil, err
+		return nil, nil
```

Comment:
It seems like this error is getting lost. Should we return an errorSeriesSet?

Reply:
yeah, that will be better. I wonder how I missed that.

Reply:
This still needs to be done